src/index.html

```diff
@@ -10,5 +10,6 @@
 <body>
 <div id="index"></div>
 <script src="static/js/libat.js"></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async></script>
```

Comment:
What is the best practice ? To keep a copy of js script locally or use remote one ? What if remote API changes ?

Reply:
It is test pr and if it will work i think we will keep source

Reply:
ok